pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -120,6 +120,7 @@ func Convert_v1_Disk_To_api_Disk(diskDevice *v1.Disk, disk *Disk, devicePerBus m
 	disk.Driver = &DiskDriver{
 		Name:  "qemu",
 		Cache: string(diskDevice.Cache),
+		IO:    string(diskDevice.IO),
```

Comment:
should we only be setting IO on the disk driver's domain xml if `diskDevice.IO != ""` ?

Reply:
Not really because the struct is set with `omitempty` for this field so if it's an empty string, it will not be dumped to the XML.
There is a test that verifies this case in this patch https://github.com/kubevirt/kubevirt/pull/3923/files#diff-406f8a4e9665b5e52c190c5815dbacd2R81